.circleci/config.yml

```diff
@@ -103,9 +104,9 @@ jobs:
           name: Run tests
           command: |
             if [[ "$CIRCLE_BRANCH" == pull/* ]]; then
-              GOEXPERIMENT=nocoverageredesign go test -v -cover -coverprofile=coverage.out ./...
+              go test -coverprofile=coverage.out ./influxdb3/
```

Comment:
Sorry, my suggestion yesterday was inexact.  I notice now that tests in `influxdb3/batching` are not being run.  I think the correct command is most likely.  

`go test -v -cover -coverprofile=coverage.out ./influxdb3/...`